papis/commands/default.py

```diff
@@ -48,6 +48,12 @@
 import click.core
 
 
+class ColoramaFormatter(logging.Formatter):
+    def format(self, record: logging.LogRecord) -> str:
+        record.msg = record.msg.format(c=colorama)
+        return super().format(record)
```

Comment:
Not sure about adding this formatter. What do you think?

It's basically meant to allow using
```
logger.info("{c.Fore.RED}My message: %s{c.Style.RESET_ALL}", msg)
```
without having to also pass in the `colorama` module. One reason I don't like it is because it's going to try to format every message, instead of just those that try to use `colorama`. Maybe adding a very short function like `_c` and then use
```
logger.info(_c("{c.Fore.RED}My message: %s{c.Style.RESET_ALL}"), msg)
```
would be better?

Either way, this is contained only in the last commit and I can revert it without worries.

Reply:
I was not aware of that, I don't think it is such aproblem
to have the formatter always format, I guess there is a way also
of disabling it.

So I'm quite ok with having the formatter like that because
in the end all messages come out formated, so we can afford the luxury
of the ergonomics.

I think it's good.

Reply:
Yeah, there are ways to disable it, so it should be ok.

Thanks for taking a look and the feedback!